src/commands/jetpack-enable-sso.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Team51\Command;
+
+use Team51\Helper\API_Helper;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Jetpack_Enable_SSO extends Command {
+    protected static $defaultName = 'jetpack:enable-sso';
```

Comment:
No strong feelings. Since we discussed the possibility of having multiple commands for Jetpack I went with the `group:command` format, but I can change it for now.